app/services/id_token_builder.rb

```diff
@@ -94,12 +85,16 @@ def determine_ial_max_acr
   end
 
   def resolved_authn_context_result
-    @resolved_authn_context_result ||= AuthnContextResolver.new(
+    @resolved_authn_context_result ||= resolved_authn_context.result
```

Comment:
since `resolve_authn_context` is now memoized, we probably don't need to memoize this?
```suggestion
    resolved_authn_context.result
```
and maybe we could even inline this whole method?

Reply:
good call, fixed in https://github.com/18F/identity-idp/pull/11335/commits/e68a5bdaf8791577a530dd9342bbb7748816697c